server/docs/gpu/api-transcription.md

```diff
@@ -0,0 +1,178 @@
+## Reflector GPU Transcription API (Parakeet)
+
+This document describes the GPU transcription API deployed on Modal.com using NVIDIA Parakeet. The API surface and response shapes are OpenAI/Whisper-compatible. If desired, you can switch to a Whisper deployment by changing the base URL; no client changes are required.
+
```

Comment:
The goal of the document was to have specification that both whisper/parakeet modal implementation respect/adhere, but also for any later implementation. 

I started a pytest script (https://github.com/Monadical-SAS/reflector/blob/main/server/tests/test_gpu_modal_transcript.py) that the idea is if you configure your variable correctly in your env, you could test the endpoint and ensure it adhere the specification. That would help for any new implementation (especially the self-hosted).